### ISSUE TITLE
Unable to create a new ignite app 

### ISSUE THREAD

**Issue by jjf21** [infinitered/ignite#1550]
What's going on?
I'm unable to create a new ignite app no matter what template I use.

Steps to reproduce:
- ignite new surf --debug
- template: Bower
- Expo: yes (also tried without, gives me another error)

Errors trace:
```
  Error: Command failed with exit code 2: ignite add ignite-bowser --debug

  - error.js:56 makeError
    [global]/[gluegun]/[execa]/lib/error.js:56:11

  - index.js:114 handlePromise
    [global]/[gluegun]/[execa]/index.js:114:26

  - task_queues.js:97 processTicksAndRejections
    internal/process/task_queues.js:97:5



      There were errors while generating the project. Run with --debug to see verbose output.

an error occured while installing ignite-bowser boilerplate.
Error: Command failed with exit code 2: ignite add ignite-bowser --debug
    at makeError (/Users/Carl/.config/yarn/global/node_modules/gluegun/node_modules/execa/lib/error.js:56:11)
    at handlePromise (/Users/Carl/.config/yarn/global/node_modules/gluegun/node_modules/execa/index.js:114:26)
    at processTicksAndRejections (internal/process/task_queues.js:97:5) {
  command: 'ignite add ignite-bowser --debug',
  exitCode: 2,
  signal: undefined,
  signalDescription: undefined,
  stdout: '\u001b[90m09:40:41\u001b[39m \u001b[35m[ignite]\u001b[39m running add command\n' +
    '\u001b[90m09:40:41\u001b[39m \u001b[35m[ignite]\u001b[39m installing /Users/Carl/Code/side-project/surf/surf/node_modules/ignite-bowser from source npm\n' +
    '\u001b[90m09:40:42\u001b[39m \u001b[35m[ignite]\u001b[39m ignite-bowser 5.0.3 on npm.\n' +
    '\u001b[90m09:40:42\u001b[39m \u001b[35m[ignite]\u001b[39m yarn add ignite-bowser --dev\n' +
    '\u001b[31m----------\u001b[39m\n' +
    '\u001b[31mCommand failed: yarn add ignite-bowser --dev\u001b[39m\n' +
    '\u001b[31m(node:90269) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.\u001b[39m\n' +
    '\u001b[31mwarning reactotron-react-native@4.2.0: This should have been a major version bump\u001b[39m\n' +
    '\u001b[31mwarning @storybook/addon-storyshots > @storybook/core > @storybook/ui > @storybook/components > popper.js@1.16.1: You can find the new Popper v2 at @popperjs/core, this package is dedicated to the legacy v1\u001b[39m\n' +
    '\u001b[31mwarning @storybook/addon-storyshots > @storybook/core > react-dev-utils > fork-ts-checker-webpack-plugin > chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.\u001b[39m\n' +
    '\u001b[31mwarning @storybook/addon-storyshots > @storybook/core > webpack > watchpack > chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.\u001b[39m\n' +
    '\u001b[31mwarning @storybook/addon-storyshots > @storybook/core > @storybook/ui > @storybook/components > react-popper-tooltip > react-popper > popper.js@1.16.1: You can find the new Popper v2 at @popperjs/core, this package is dedicated to the legacy v1\u001b[39m\n' +
    '\u001b[31mwarning jest > jest-cli > jest-config > jest-environment-jsdom > jsdom > request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142\u001b[39m\n' +
    '\u001b[31mwarning jest-expo > jest > jest-cli > jest-config > jest-environment-jsdom > jsdom > request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142\u001b[39m\n' +
    '\u001b[31mwarning jest-expo > jest > jest-cli > jest-config > jest-environment-jsdom > jsdom > left-pad@1.3.0: use String.prototype.padStart()\u001b[39m\n' +
    '\u001b[31mwarning solidarity > gluegun > enquirer > set-value@1.0.0: Critical bug fixed in v3.0.1, please upgrade to the latest version.\u001b[39m\n' +
    '\u001b[31mwarning solidarity > gluegun > prompt-autocompletion > prompt-choices > set-value@0.4.3: Critical bug fixed in v3.0.1, please upgrade to the latest version.\u001b[39m\n' +
    '\u001b[31mwarning " > reactotron-mst@3.1.3" has unmet peer dependency "reactotron-core-client@^2.5.0".\u001b[39m\n' +
    '\u001b[31mwarning "reactotron-react-native > react-native-flipper@0.34.0" has incorrect peer dependency "react-native@^0.62.0".\u001b[39m\n' +
    '\u001b[31mwarning " > @react-navigation/stack@5.3.3" has unmet peer dependency "@react-native-community/masked-view@>= 0.1.0".\u001b[39m\n' +
    '\u001b[31mwarning " > @react-navigation/stack@5.3.3" has unmet peer dependency "react-native-gesture-handler@>= 1.0.0".\u001b[39m\n' +
    '\u001b[31mwarning "@storybook/addon-storyshots > @storybook/core@5.3.17" has unmet peer dependency "babel-loader@^7.0.0 || ^8.0.0".\u001b[39m\n' +
    '\u001b[31mwarning "@storybook/addon-storyshots > @storybook/core > @storybook/router > @reach/router > create-react-context@0.3.0" has unmet peer dependency "prop-types@^15.0.0".\u001b[39m\n' +
    '\u001b[31mwarning "@storybook/react-native > @emotion/native > @emotion/primitives-core@10.0.27" has unmet peer dependency "@emotion/core@^10.0.27".\u001b[39m\n' +
    '\u001b[31mwarning " > @storybook/react-native-server@5.3.17" has unmet peer dependency "babel-loader@^7.0.0 || ^8.0.0".\u001b[39m\n' +
    '\u001b[31mwarning "@storybook/react-native-server > ws@7.3.0" has unmet peer dependency "bufferutil@^4.0.1".\u001b[39m\n' +
    '\u001b[31mwarning "@storybook/react-native-server > ws@7.3.0" has unmet peer dependency "utf-8-validate@^5.0.2".\u001b[39m\n' +
    '\u001b[31mwarning "jest > jest-cli > jest-config > jest-environment-jsdom > jsdom@15.2.1" has unmet peer dependency "canvas@^2.5.0".\u001b[39m\n' +
    '\u001b[31merror Command failed with exit code 1.\u001b[39m\n' +
    '\u001b[31m\u001b[39m\n' +
    '\u001b[31m----------\u001b[39m',
  stderr: '- adding \u001b[36mignite-bowser\u001b[39m\n' +
    '✖ \u001b[31mignite-bowser\u001b[39m was not able to be installed. Is it a valid NPM module?',
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false
}
error installing boilerplate
```
---
`ignite doctor` results:
```
System
  platform           darwin
  arch               x64
  cpu                4 cores        Intel(R) Core(TM) i5-5257U CPU @ 2.70GHz
  directory          side-project   /Users/Carl/Code/side-project

JavaScript
  node               12.16.3      /usr/local/bin/node
  npm                6.14.4       /usr/local/bin/npm
  yarn               1.3.2        /usr/local/bin/yarn

Ignite
  ignite-cli         3.5.1        /Users/Carl/.npm-packages/bin/ignite
  ignite src         build        /Users/Carl/.config/yarn/global/node_modules/ignite-cli/build
  generators         {}

Android
  java               9.0.4        /usr/bin/java
  android home       -            undefined

iOS
  xcode              10.0
  Error: Command failed: pod --version
  /Users/Carl/.rbenv/versions/2.3.4/bin/ruby: invalid option -:  (-h will show valid options) (RuntimeError)

  - child_process.js:303 ChildProcess.exithandler
    child_process.js:303:12

  - events.js:310 ChildProcess.emit
    events.js:310:20

  - child_process.js:1021 maybeClose
    internal/child_process.js:1021:16

  - child_process.js:286 Process.ChildProcess._handle.onexit
    internal/child_process.js:286:5


/Users/Carl/.config/yarn/global/node_modules/gluegun/build/index.js:13
    throw up;
    ^

Error: Command failed: pod --version
/Users/Carl/.rbenv/versions/2.3.4/bin/ruby: invalid option -:  (-h will show valid options) (RuntimeError)

    at ChildProcess.exithandler (child_process.js:303:12)
    at ChildProcess.emit (events.js:310:20)
    at maybeClose (internal/child_process.js:1021:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:286:5) {
  killed: false,
  code: 1,
  signal: null,
  cmd: 'pod --version',
  stderr: '/Users/Carl/.rbenv/versions/2.3.4/bin/ruby: invalid option -:  (-h will show valid options) (RuntimeError)\n'
}
```

I've had the same error before running `sudo gem install cocoapods` which give the error in ignite doctor

**Comment by pntgupta** [comment]
Facing same issue.

**Comment by bitbreakr** [comment]
Same here

**Comment by kevinvangelder** [comment]
Looks like the error is `can't find ignite-bowser`. Occasionally we see this issue when people have custom .npmrc that prevent NPM/yarn from looking at the global NPM registry. You also have an error where the `pod` command isn't found. If you use a ruby switcher like `chruby` make sure you've selected a version where cocoapods is available.

**Comment by banerjed** [comment]
Having the exact same issue

✔ Which boilerplate would you like to use? · ignite-bowser
✔ using the Infinite Red Bowser boilerplate
✔ Would you like to use Expo on this project?


        Why Expo? Expo (https://expo.io) is the fastest way to get started.
        However, Expo support is experimental at this time. If unsure, select No and we'll install the traditional route.
        Additionally, using Expo means you won't be able to add any custom native modules.

       (y/N) · true
  
  We'll initiate your app using Expo. Please note that you won't be able
  to use native modules unless you "eject".
  
  More info here: https://docs.expo.io/versions/latest/expokit/eject/
  
✔ initialized Expo app  in 197.13s




      There were errors while generating the project. Run with --debug to see verbose output.
    
an error occured while installing ignite-bowser boilerplate.
Error: Command failed with exit code 2: ignite add ignite-bowser 
    at makeError (/Users/dhritimanbanerjee/.config/yarn/global/node_modules/gluegun/node_modules/execa/lib/error.js:56:11)
    at handlePromise (/Users/dhritimanbanerjee/.config/yarn/global/node_modules/gluegun/node_modules/execa/index.js:114:26)
    at processTicksAndRejections (internal/process/task_queues.js:97:5) {
  command: 'ignite add ignite-bowser ',
  exitCode: 2,
  signal: undefined,
  signalDescription: undefined,
  stdout: '\u001b[31m----------\u001b[39m\n' +
    '\u001b[31mCommand failed: yarn add ignite-bowser --dev\u001b[39m\n' +
    '\u001b[31mwarning reactotron-react-native@4.2.0: This should have been a major version bump\u001b[39m\n' +
    '\u001b[31mwarning @storybook/addon-storyshots > @storybook/core > react-dev-utils > fork-ts-checker-webpack-plugin > chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.\u001b[39m\n' +
    '\u001b[31mwarning @storybook/addon-storyshots > @storybook/core > webpack > watchpack > watchpack-chokidar2 > chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.\u001b[39m\n' +
    '\u001b[31mwarning @storybook/addon-storyshots > @storybook/core > @storybook/ui > @storybook/components > popper.js@1.16.1: You can find the new Popper v2 at @popperjs/core, this package is dedicated to the legacy v1\u001b[39m\n' +
    '\u001b[31mwarning @storybook/addon-storyshots > @storybook/core > @storybook/ui > @storybook/components > react-popper-tooltip > react-popper > popper.js@1.16.1: You can find the new Popper v2 at @popperjs/core, this package is dedicated to the legacy v1\u001b[39m\n' +
    '\u001b[31mwarning jest > jest-cli > jest-config > jest-environment-jsdom > jsdom > request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142\u001b[39m\n' +
    '\u001b[31mwarning jest-expo > jest > jest-cli > jest-config > jest-environment-jsdom > jsdom > request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142\u001b[39m\n' +
    '\u001b[31mwarning jest-expo > jest > jest-cli > jest-config > jest-environment-jsdom > jsdom > left-pad@1.3.0: use String.prototype.padStart()\u001b[39m\n' +
    '\u001b[31mwarning solidarity > gluegun > enquirer > set-value@1.0.0: Critical bug fixed in v3.0.1, please upgrade to the latest version.\u001b[39m\n' +
    '\u001b[31mwarning solidarity > gluegun > prompt-autocompletion > prompt-choices > set-value@0.4.3: Critical bug fixed in v3.0.1, please upgrade to the latest version.\u001b[39m\n' +
    '\u001b[31mwarning solidarity > gluegun > enquirer > prompt-question > prompt-choices > set-value@1.0.0: Critical bug fixed in v3.0.1, please upgrade to the latest version.\u001b[39m\n' +
    '\u001b[31mwarning " > reactotron-mst@3.1.3" has unmet peer dependency "reactotron-core-client@^2.5.0".\u001b[39m\n' +
    '\u001b[31mwarning " > @react-navigation/stack@5.6.2" has unmet peer dependency "@react-native-community/masked-view@>= 0.1.0".\u001b[39m\n' +
    '\u001b[31mwarning " > @react-navigation/stack@5.6.2" has unmet peer dependency "react-native-gesture-handler@>= 1.0.0".\u001b[39m\n' +
    '\u001b[31mwarning "@storybook/addon-storyshots > @storybook/core@5.3.17" has unmet peer dependency "babel-loader@^7.0.0 || ^8.0.0".\u001b[39m\n' +
    '\u001b[31mwarning "@storybook/react-native-server > @storybook/api@5.3.17" has unmet peer dependency "regenerator-runtime@*".\u001b[39m\n' +
    '\u001b[31mwarning " > @storybook/react-native-server@5.3.17" has unmet peer dependency "babel-loader@^7.0.0 || ^8.0.0".\u001b[39m\n' +
    '\u001b[31mwarning " > jest-expo@36.0.1" has incorrect peer dependency "react@~16.9.0".\u001b[39m\n' +
    '\u001b[31merror Command failed with exit code 1.\u001b[39m\n' +
    '\u001b[31m\u001b[39m\n' +
    '\u001b[31m----------\u001b[39m',
  stderr: '- adding \u001b[36mignite-bowser\u001b[39m\n' +
    '✖ \u001b[31mignite-bowser\u001b[39m was not able to be installed. Is it a valid NPM module?',
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false
}
error installing boilerplate

**Comment by hbxz** [comment]
> Having the exact same issue. NOTE: it does seem that the app got generated, in spite of the errors
> 
> ✔ Which boilerplate would you like to use? · ignite-bowser
> ✔ using the Infinite Red Bowser boilerplate
> ✔ Would you like to use Expo on this project?
> 
> ```
>     Why Expo? Expo (https://expo.io) is the fastest way to get started.
>     However, Expo support is experimental at this time. If unsure, select No and we'll install the traditional route.
>     Additionally, using Expo means you won't be able to add any custom native modules.
> 
>    (y/N) · true
> ```
> 
> We'll initiate your app using Expo. Please note that you won't be able
> to use native modules unless you "eject".
> 
> More info here: https://docs.expo.io/versions/latest/expokit/eject/
> 
> ✔ initialized Expo app in 197.13s
> 
> ```
>   There were errors while generating the project. Run with --debug to see verbose output.
> ```
> 
> an error occured while installing ignite-bowser boilerplate.
> Error: Command failed with exit code 2: ignite add ignite-bowser
> at makeError (/Users/dhritimanbanerjee/.config/yarn/global/node_modules/gluegun/node_modules/execa/lib/error.js:56:11)
> at handlePromise (/Users/dhritimanbanerjee/.config/yarn/global/node_modules/gluegun/node_modules/execa/index.js:114:26)
> at processTicksAndRejections (internal/process/task_queues.js:97:5) {
> command: 'ignite add ignite-bowser ',
> exitCode: 2,
> signal: undefined,
> signalDescription: undefined,
> stdout: '\u001b[31m----------\u001b[39m\n' +
> '\u001b[31mCommand failed: yarn add ignite-bowser --dev\u001b[39m\n' +
> '\u001b[31mwarning reactotron-react-native@4.2.0: This should have been a major version bump\u001b[39m\n' +
> '\u001b[31mwarning @storybook/addon-storyshots > @storybook/core > react-dev-utils > fork-ts-checker-webpack-plugin > chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.\u001b[39m\n' +
> '\u001b[31mwarning @storybook/addon-storyshots > @storybook/core > webpack > watchpack > watchpack-chokidar2 > chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.\u001b[39m\n' +
> '\u001b[31mwarning @storybook/addon-storyshots > @storybook/core > @storybook/ui > @storybook/components > popper.js@1.16.1: You can find the new Popper v2 at @popperjs/core, this package is dedicated to the legacy v1\u001b[39m\n' +
> '\u001b[31mwarning @storybook/addon-storyshots > @storybook/core > @storybook/ui > @storybook/components > react-popper-tooltip > react-popper > popper.js@1.16.1: You can find the new Popper v2 at @popperjs/core, this package is dedicated to the legacy v1\u001b[39m\n' +
> '\u001b[31mwarning jest > jest-cli > jest-config > jest-environment-jsdom > jsdom > request@2.88.2: request has been deprecated, see https://github.com/[request/request/issues/3142](https://github.com/request/request/issues/3142)\u001b[39m\n' +
> '\u001b[31mwarning jest-expo > jest > jest-cli > jest-config > jest-environment-jsdom > jsdom > request@2.88.2: request has been deprecated, see https://github.com/[request/request/issues/3142](https://github.com/request/request/issues/3142)\u001b[39m\n' +
> '\u001b[31mwarning jest-expo > jest > jest-cli > jest-config > jest-environment-jsdom > jsdom > left-pad@1.3.0: use String.prototype.padStart()\u001b[39m\n' +
> '\u001b[31mwarning solidarity > gluegun > enquirer > set-value@1.0.0: Critical bug fixed in v3.0.1, please upgrade to the latest version.\u001b[39m\n' +
> '\u001b[31mwarning solidarity > gluegun > prompt-autocompletion > prompt-choices > set-value@0.4.3: Critical bug fixed in v3.0.1, please upgrade to the latest version.\u001b[39m\n' +
> '\u001b[31mwarning solidarity > gluegun > enquirer > prompt-question > prompt-choices > set-value@1.0.0: Critical bug fixed in v3.0.1, please upgrade to the latest version.\u001b[39m\n' +
> '\u001b[31mwarning " > reactotron-mst@3.1.3" has unmet peer dependency "reactotron-core-client@^2.5.0".\u001b[39m\n' +
> '\u001b[31mwarning " > @react-navigation/stack@5.6.2" has unmet peer dependency "@react-native-community/masked-view@>= 0.1.0".\u001b[39m\n' +
> '\u001b[31mwarning " > @react-navigation/stack@5.6.2" has unmet peer dependency "react-native-gesture-handler@>= 1.0.0".\u001b[39m\n' +
> '\u001b[31mwarning "@storybook/addon-storyshots > @storybook/core@5.3.17" has unmet peer dependency "babel-loader@^7.0.0 || ^8.0.0".\u001b[39m\n' +
> '\u001b[31mwarning "@storybook/react-native-server > @storybook/api@5.3.17" has unmet peer dependency "regenerator-runtime@*".\u001b[39m\n' +
> '\u001b[31mwarning " > @storybook/react-native-server@5.3.17" has unmet peer dependency "babel-loader@^7.0.0 || ^8.0.0".\u001b[39m\n' +
> '\u001b[31mwarning " > jest-expo@36.0.1" has incorrect peer dependency "react@~16.9.0".\u001b[39m\n' +
> '\u001b[31merror Command failed with exit code 1.\u001b[39m\n' +
> '\u001b[31m\u001b[39m\n' +
> '\u001b[31m----------\u001b[39m',
> stderr: '- adding \u001b[36mignite-bowser\u001b[39m\n' +
> '✖ \u001b[31mignite-bowser\u001b[39m was not able to be installed. Is it a valid NPM module?',
> failed: true,
> timedOut: false,
> isCanceled: false,
> killed: false
> }
> error installing boilerplate

Same issue here

**Comment by flexbox** [comment]
Hey @bitbreakr 
Do you still have the issue 😉
